knimin/lib/tests/test_data_access.py

```diff
@@ -1664,6 +1664,27 @@ def test_get_targeted_plate_list(self):
                 'date': datetime.date.today(), 'num_samples': 4}]
         self.assertItemsEqual(db.get_targeted_plate_list(), exp)
 
+    def test_get_quantified_targeted_plate_list(self):
+        targeted_plate_ids = self._create_test_data_targeted_plate()
+        self.assertItemsEqual(db.get_quantified_targeted_plate_list(), [])
+
+        vals = np.zeros((8, 12))
+        db.quantify_targeted_plate(targeted_plate_ids[0],
+                                   'raw_concentration', vals)
+        db.quantify_targeted_plate(targeted_plate_ids[0],
+                                   'mod_concentration', vals)
```

Comment:
Discussed with the wet lab and that is correct. I was missing one step and that's why I was not adjusting the volume. Now it should be correct.